recipes/wineditline/all/CMakeLists.txt

```diff
@@ -1,8 +1,17 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
+project(WinEditLine C)
 
-project(WinEditLineWrapper C)
+add_library(edit
```

Comment:
The upstream uses different names. Why not keeping the upstream cmake instead?

https://sourceforge.net/p/mingweditline/code/HEAD/tree/mingweditline/trunk/src/CMakeLists.txt

Reply:
ping @valgur

Reply:
Don't remember at all anymore why I ignored the project CMakeLists.txt. I'll look into it.

Reply:
Hi @valgur, thanks for your contributions, any update on this?

Reply:
You can use the original CMake with some additions, I'll provide them below. With these additions, you don't need to have any custom CMakeLists inside the package at all